packages/cli/src/checks/EXTERNAL_MISMATCH.ts

```diff
@@ -25,9 +25,13 @@ export default makeCheck<ErrorType>({
         for (let depName in deps) {
           let range = deps[depName];
           let mostCommonRange = mostCommonRangeMap.get(depName);
+          const allowedVersions =
+            options.allowedDependencyVersions &&
+            options.allowedDependencyVersions[depName];
           if (
             mostCommonRange !== undefined &&
             mostCommonRange !== range &&
+            !(allowedVersions && allowedVersions.includes(range)) &&
```

Comment:
Just to summarise, it'll error anything that isn't most common or in allowedVersions? When fixing it'll fix to most common?

Reply:
yep correct